guides/doc-Managing_Hosts/master.adoc

```diff
@@ -33,6 +33,8 @@ include::topics/using_report_templates.adoc[leveloffset=+1]
 
 include::topics/Configuring_Host_Collections.adoc[]
 
```

Comment:
Can you add an ifndef for Satellite, pretty please :D

Reply:
Added.